internal/pkg/build/sources/conveyorPacker_library.go

```diff
@@ -70,11 +76,36 @@ func (cp *LibraryConveyorPacker) Get(b *types.Bundle) (err error) {
 		}
 	}
 
+	if !cp.AllowUnauthenticated {
+		// check if the base container is signed
+		imageSigned, err := signing.IsSigned(imagePath, "https://keys.sylabs.io", 0, false, cp.AuthToken, cp.LocalVerify, true)
+		if err != nil {
+			sylog.Warningf("%v", err)
```

Comment:
@WestleyK What are the possible errors here? It its only that it can't be verified, then i think the flow on line 89 covers this case

Reply:
There are two common errors:

```bash
WARNING: unable to verify container: error while searching for signature blocks: no signatures found for system partition

WARNING: unable to verify container: unable to verify container: openpgp: signature made by unknown entity
```

Reply:
So it looks like the first case is the only case where you return `false` indicating its not signed, why have an error when the boolean value already gives you the same information and this is expected if it is indeed not signed?